sites/footers/deprecated/footers-v2-en.html

```diff
@@ -0,0 +1,118 @@
+---
+{
+"layout": false,
+"title": "Content page including complete footer version 2.0 (deprecated)",
+"language": "en",
+"breadcrumbs": [
+	{ "title": "Footer", "link": "sites/footers/footers-en.html" }
+],
+"altLangPage": "footers-v2-fr.html",
+"secondlevel": false,
+"dateModified": "2022-06-17",
+"share": "true"
+}
+---
+
+{%- include variable-core.liquid -%}
+{%- capture page-title -%}
+{%- if page.title -%}
+{{ page.title }}
+{%- else -%}
+Page untitled
+{%- endif -%}
+{%- endcapture -%}
+<!DOCTYPE html>
+<html class="no-js" lang="{{ i18nText-lang | default: 'en' }}" dir="{{ i18nText-langDir | default: 'ltr' }}">
+
+<head>
+	<meta charset="utf-8">
+	{% include license.html %}
+	<title>{{ page-title }} - {{ i18nText-siteTitle }}</title>
+	<meta content="width=device-width, initial-scale=1" name="viewport">
+	{% include metadata.html %}
+	{% include resources-inc/head.html %}
+</head>
+
+<body {% if page.pageclass %}class="{{ page.pageclass }}" {% endif %}vocab="http://schema.org/" typeof="WebPage">
+	{%- if page.archived -%}
+	{% include headers-includes/archive.html %}
+	{%- endif -%}
+	{% include skiplinks/skiplinks.html %}
+	{% include header/header.html %}
+	<main class="container" property="mainContentOfPage" resource="#wb-main" typeof="WebPageElement">
+		<h1 id="wb-cont" property="name">{{ page-title }}</h1>
+		<p>This is version 2.0 (deprecated) of the page footer including the "About government" section and all utility links from "About this site" section.</p>
+		<p>Updated to use the <code>colcount-xs-2</code> class instead of <code>colcount-sm-2</code>.</p>
+		{% include page-details/footer.html %}
+	</main>
+	<footer id="wb-info">
+		{% unless page.noFooterAbout %}
+		<div class="landscape">
+			<nav class="container wb-navcurr">
```

Comment:
The landscape image are cut because of the new padding value,
![image](https://user-images.githubusercontent.com/1676030/194203253-3cdf81a6-ab72-46f3-882f-2a5b05a04119.png)



Let explore how we can apply a transitory style while implementing the markup change.

**Option 1** - Set a min height
```css
#wb-info .landscape nav {
  min-height: 180px;
}
```
![image](https://user-images.githubusercontent.com/1676030/194202047-c0c9d902-561f-4ae9-b67c-99dcde4fe193.png)

**Option 2** - Apply corrective padding value
```
#wb-info .landscape nav h2.wb-inv + ul {
  padding-bottom: 1em;
  padding-top: 0.6em;
}
```
![image](https://user-images.githubusercontent.com/1676030/194202652-800540b8-6794-47eb-a0b0-a0b4b72e5545.png)

The option 2 seems to best reproduce the current design without cropping the parliament flag in the background image. I didn't tested if that solution impact somehow the new design. @GormFrank what do you think?

Reply:
I propose the following:

```
#wb-info .landscape .wb-navcurr {
  padding-bottom: 2em;
  padding-top: 2em;
}
```

This resets the CSS that was already there. `.wb-navcurr` is removed in the new footer as the class was not used, so we can leverage it in this case.

@duboisp What do you think?